src/styles/blocks/recommended.scss

```diff
@@ -0,0 +1,61 @@
+.recommended {
+  &__item {
+    margin-bottom: 48px;
+
+    &:last-child {
+      margin-bottom: 0;
+    }
+  }
+
+  @include onTablet {
+    margin-bottom: 40px;
+
+    &:last-child {
+      margin-bottom: 0;
+    }
+  }
+
+  @include onDesktop {
+    margin-bottom: 0;
+  }
+}
+
+.recommend {
+  &__photo-container {
+    display: grid;
+    grid-template-columns: repeat(4, 1fr);
+    column-gap: 24px;
+
+    margin-bottom: 32px;
+  }
+
+  &__photo {
+    grid-column: 2 / 4;
+
+    width: 100%;
+  }
```

Comment:
There should be hover effects (checklist #10)

These pictures are too small on mobile view. Try to remove grid display for .recommend__photo-container on mobile.